scripts/zkai_stwo_statement_envelope_benchmark.py

```
@@ -0,0 +1,717 @@
+        raise StwoEnvelopeError(f"Stwo verify-stark verifier rejected: {output}")
+    if "verified_stark: true" not in output:
+        raise StwoEnvelopeError(f"Stwo verify-stark did not report verified_stark: true: {output}")
+
+
+def verify_statement_envelope(
+    envelope: dict[str, Any],
+    *,
+    external_verify: Callable[[dict[str, Any]], None] = stwo_verify,
+) -> None:
+    if envelope.get("schema") != ENVELOPE_SCHEMA:
+        raise StwoEnvelopeError("unsupported envelope schema")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)